dev-requirements.txt

```
@@ -1,7 +1,12 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+
+git+https://github.com/dbt-labs/dbt-core.git@ct-808-grant_adapter_tests#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@ct-808-grant_adapter_tests#egg=dbt-tests-adapter&subdirectory=tests/adapter
```

just a heads up @emmyoop ct-808 has been merged into my [core pr](https://github.com/dbt-labs/dbt-core/pull/5369) if you want to keep these pointed at a targeted version 

```suggestion
git+https://github.com/dbt-labs/dbt-core.git@ct-660-grant-sql#egg=dbt-core&subdirectory=core
git+https://github.com/dbt-labs/dbt-core.git@ct-660-grant-sql#egg=dbt-tests-adapter&subdirectory=tests/adapter
```
should work till we merge into main.